locale/ru/LC_MESSAGES/doc.po

```diff
@@ -183,14 +183,14 @@ msgstr "Для создания смешанной резервной копии
 #: ../doc/1.10/book/admin/backups.rst:124
 msgid ""
 "Issue :ref:`box.backup.start() <admin-backups-backup_start>` on the "
-":ref:`administrative console <admin-security>`. This will suspend garbage"
-" collection till the next ``box.backup.stop()`` and will return a list of"
-" files to back up."
+":ref:`administrative console <admin-security>`. This will
+" return a list of files to back up and suspend garbage"
+" collection of them till the next ``box.backup.stop()``."
```

Comment:
... collection FOR them...

Reply:
Is it matter for translation whether i change msgid in po files if they are generated as we discussed?